specification/trace/semantic_conventions/cloudevents.md

```diff
@@ -0,0 +1,51 @@
+# CloudEvents
+
+This document defines how to apply semantic conventions when instrumenting code that uses [CloudEvents](https://cloudevents.io/).
+These semantic conventions are general and not tied to any protocol used to send/receive the events.
+
+**Status**: [Experimental](../../document-status.md)
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Span name](#span-name)
+- [Span kind](#span-kind)
+- [Attributes](#attributes)
+
+<!-- tocstop -->
+
+## Span name
+
+The span name SHOULD be set to the `cloudevents.` prefix, followed by the event type in the following format:
+
+```
+cloudevents.<event type name>
```

Comment:
I had initially the span name as `cloudevents.<event type> <operation name>` where the `operation name` followed the ones listed for messaging systems (`send`, `receive` and `process`). But I felt that adding it here now that this is completely separated from messaging would cause confusion so I left it out. What do you think? 

If I add it here, then I also need to list the operation names and it will be somewhat duplicated from the `messaging.md` file. Since they made it very clear from the PR discussions and the SIG meeting that CloudEvents are NOT a messaging system I feel that adding this will trigger again the discussions above, even though in my view as a user it makes sense.

Reply:
If we don't want to distinguish receive and process then the span kind will be sufficient to tell which end of the operation is represented here.

Reply:
Yeah I also think so. With the span kind it should be sufficient to tell which end it is.